backend/src/test/java/com/woowacourse/kkogkkog/application/MemberServiceTest.java

```diff
@@ -37,4 +38,17 @@ void save_fail_duplicatedEmail() {
         assertThatThrownBy(() -> memberService.save(memberCreateRequest))
             .isInstanceOf(MemberDuplicatedEmail.class);
     }
+
+    @Test
+    @DisplayName("회원가입된 회원들의 정보를 조회할 수 있다.")
+    void findAll() {
+        MemberCreateRequest memberCreateRequest1 = new MemberCreateRequest("email1@gmail.com", "password1234!", "nickname1");
+        MemberCreateRequest memberCreateRequest2 = new MemberCreateRequest("email2@gmail.com", "password1234!", "nickname2");
+        memberService.save(memberCreateRequest1);
+        memberService.save(memberCreateRequest2);
+
```

Comment:
변수명 짓기 애매하면 그대로 받아서 사용하는 것도 하나의 방법 같아요.

```suggestion
        memberService.save(new MemberCreateRequest("email1@gmail.com", "password1234!", "nickname1"));
        memberService.save(new MemberCreateRequest("email2@gmail.com", "password1234!", "nickname2"));
```

Reply:
이부분은 2주차 스프린트 종료 후 fixture 재논의 시점에서 변경될 것 같아서 그대로 두었습니다. 목록에 정리해 둘게요~!